bodhi/client/__init__.py

```diff
@@ -272,6 +272,28 @@ def composes():
     pass  # pragma: no cover
 
 
+@composes.command(name='info')
+@handle_errors
+@click.option('--release', required=True, help='The name of the release that is being composed.')
+@click.option('--request', default='testing',
```

Comment:
It's a bit arbitrary, but it seems like users might be more likely to want to know when a stable compose is done than a testing compose, so I'd recommend defaulting to stable here.

```suggestion
@click.option('--request', default='stable',
```